doc/install.md

```diff
@@ -20,8 +21,29 @@ If you do not have a cluster yet, create one by using one of the following provi
 * [kubeadm](https://kubernetes.io/docs/admin/kubeadm/)
 * [minikube](https://minikube.sigs.k8s.io/)
 
-## Install release v0.19.8 and use Coscheduling
-
+## Install release v0.19.8 as a second scheduler and use Coscheduling
+The quickest way to try scheduler-plugins is to install it using helm chart as a second scheduler.
+You can find the demo chart in [manifests/install/charts](../manifests/install/charts).
```

Comment:
Q: Is there a lot of effort to add a new helm repo?

Reply:
Not difficult, but where should I place the helm repo?Is this a common rule tells where to place helm chart to ?